docs/reference/release-notes/highlights.asciidoc

```diff
@@ -25,6 +25,19 @@ Other versions:
 
 // tag::notable-highlights[]
 [discrete]
+[[indexing-speed-improvement]]
+===  Indexing speed improvement
+
+{es} 7.10 improves the indexing speed by 5-20%, depending on your workload.
```

Comment:
Maybe say "up to 20%" without providing a lower bound. As you highlighted later, scenarios with full-text indexing or ingest might not notice a speedup.